src/features/nft/portfolio/PortfolioPage.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { TNFT } from 'features/nft/Nft.types'
+import dayjs from 'dayjs'
+import Breadcrumbs from './Breadcrumbs'
+import Header from './Header'
+import Image from './image/Image'
+import Info from 'features/nft/portfolio/info/Info'
+import Description from 'features/nft/portfolio/description/Description'
+import Comments from 'features/nft/portfolio/comments/Comments'
```

Comment:
Can we use relative paths here and other places?

Reply:
fixed!